pkg/api-server/config_ws_test.go

```diff
@@ -336,6 +336,10 @@ var _ = Describe("Config WS", func() {
               "generateZoneToken": {
                 "users": ["mesh-system:admin"],
                 "groups": ["mesh-system:admin"]
+              },
+              "getConfigDump": {
+                "users": ["mesh-system:anonymous","mesh-system:admin"],
+                "groups": ["mesh-system:unauthenticated","mesh-system:admin"]
```

Comment:
I think it should be `mesh-system:unauthenticated` and `mesh-system:authenticated`. Otherwise if someone other that admin logs in into kumactl they won't be able to execute the command